src/slic3r/Utils/PresetUpdater.cpp

```diff
@@ -1358,8 +1358,12 @@ void PresetUpdater::sync(std::string http_url, std::string language, std::string
         }
 		if (p->cancel)
 			return;
-        this->p->sync_plugins(http_url, plugin_version);
-        this->p->sync_printer_config(http_url);
+
+        // orca: http_url is empty string in stealth_mode
+        if (http_url.size() > 0) {
+            this->p->sync_plugins(http_url, plugin_version);
```

Comment:
Currently plugin is no controlled by stealth flag, it's enabled networking option that controls it.
I'm afraid disabling plugin downloading will confuse user.
Better don't do it.

Stealth flag mainly disable HMS